pkg/apis/certmanager/v1alpha2/types_certificate.go

```diff
@@ -145,6 +150,21 @@ type CertificateSpec struct {
 	KeyEncoding KeyEncoding `json:"keyEncoding,omitempty"`
 }
 
+type X509DistinguishedName struct {
```

Comment:
Perhaps this can be `X509Subject` instead?

Reply:
Also can you add a comment above this type, as well as above each field within the struct? Doing so means that a user can run `kubectl explain certificate.spec.subject` and see a descriptive list of all supported fields 😄